src/inference/include/ie/gpu/gpu_config.hpp

```diff
@@ -76,48 +65,6 @@ namespace GPUConfigParams {
 #define DECLARE_GPU_CONFIG_KEY(name)   DECLARE_CONFIG_KEY(GPU_##name)
 #define DECLARE_GPU_CONFIG_VALUE(name) DECLARE_CONFIG_VALUE(GPU_##name)
 
-/**
- * @brief This key instructs the GPU plugin to use two priorities of GPU configuration as follows:
- * • OpenCL queue priority hint as defined in https://www.khronos.org/registry/OpenCL/specs/opencl-2.1-extensions.pdf,
- *      it has 4 types of levels: High, Med, Low, and Default. the default is Default
- * • Host task priority which is set cpu core type of TBB affinity used in load network.
- *      this has 3 types of levels: High, LOW, and ANY. the default is ANY.
- *      it is only affected on Hybrid CPUs. if the device doesn't support Hybrid CPUs, it is set to the default.
- *
- * There are two types of setting you can choose from: Model level setting and Queue/Host Task level setting.
- * • Plugin level setting is the predefined combination of OpenCL queue priority and host task priority.
- *      It provides only two types of levels: High and Low.
- * • Queue/Host Task level setting is the combination of OpenCL Queue priority and host task priority
- *      such as GPU_QUEUE_PRIORITY_HIGH|GPU_HOST_TASK_PRIORITY_HIGH.
- *      You can set each levels of OpenCL Queue priority and host task priority directly using this setting.
- *
- * The default value of GPU_MODEL_PRIORITY is "GPI_QUEUE_PRIORITY_DEFAULT|GPU_HOST_TASK_PRIORITY_ANY".
- * The detailed option values are as follows:
- * Model priority
- * • GPUConfigParams::GPU_MODEL_PRIORITY_HIGH  - GPU_QUEUE_PRIORITY_HIGH|GPU_HOST_TASK_PRIORITY_HIGH
- * • GPUConfigParams::GPU_MODEL_PRIORITY_LOW   - GPU_QUEUE_PRIORITY_LOW|GPU_HOST_TASK_PRIORITY_LOW
- * OpenCL queue priority
- * • GPUConfigParams::GPU_QUEUE_PRIORITY_HIGH       - mapped to CL_QUEUE_PRIORITY_HIGH_KHR
- * • GPUConfigParams::GPU_QUEUE_PRIORITY_MED        - mapped to CL_QUEUE_PRIORITY_MED_KHR
- * • GPUConfigParams::GPU_QUEUE_PRIORITY_LOW        - mapped to CL_QUEUE_PRIORITY_LOW_KHR
- * • GPUConfigParams::GPI_QUEUE_PRIORITY_DEFAULT    - Not set queue priority property in cl_queue_properties
- * Host task priority
- * • GPUConfigParams::GPU_HOST_TASK_PRIORITY_HIGH   - mapped to IStreamsExecutor::Config::BIG
- * • GPUConfigParams::GPU_HOST_TASK_PRIORITY_LOW    - mapped to IStreamsExecutor::Config::LITTLE
- * • GPUConfigParams::GPU_HOST_TASK_PRIORITY_ANY    - mapped to IStreamsExecutor::Config::ANY
- */
-
-DECLARE_GPU_CONFIG_KEY(MODEL_PRIORITY);
```

Comment:
Why did you remove some code from legacy API?

Reply:
I I was not aware about the decision to keep not released keys in both APIs. This change is reverted, thank you